core/common/src/rpc/core/RpcInvocation.ts

```diff
@@ -238,11 +247,10 @@ export class RpcInvocation {
     return fulfillment;
   }
 
-  private lookupOperationFunction(implementation: RpcInterface): (...args: any[]) => any {
```

Comment:
Why is this returning a promise now? We aren’t awaiting the call to it above

Reply:
all this change does is make it clear that every RPC operation must return a Promise

Reply:
Sorry I misread the signature.